iothub_client/samples/iothub_client_sample_upload_to_blob_mb/iothub_client_sample_upload_to_blob_mb_stress.c

```diff
@@ -29,10 +29,14 @@ and removing calls to _DoWork will yield the same results. */
 #define ITERATIONS    1
 #define MAX_BLOCK_SIZE_BYTES    4194304
 
-/*Optional string with http proxy host and integer for http proxy port (Linux only)         */
+/*Optional string with http proxy host and integer for http proxy port. (Both Win and Lin when uhttp is used.)*/
```

Comment:
This is not very clear I would say.
Either it needs to be more explanatory, or don't mention the "Both Win ..."